curated_transformers/util/serde.py

```diff
@@ -18,10 +34,52 @@
 ]
 
 
+class ModelCheckpointType(Enum):
+    """
+    Types of model checkpoints supported by Curated Transformers.
+    """
+
+    #: PyTorch checkpoint.
+    PYTORCH = 1
```

Comment:
Maybe we should be more specific here, since it is possible that we have other types in the future? Maybe `PYTORCH _LOAD` or `PYTORCH_STATE_DICT`? Also, I'd start numbering at 0, C-style.

Reply:
Renamed to `PYTORCH_STATE_DICT` and renumbered.